src/config/Config.zig

```diff
@@ -2153,28 +2147,14 @@ keybind: Keybinds = .{},
 ///    more subtle border.
 ///
 /// Changing this value at runtime will only affect new windows.
-@"adw-toolbar-style": AdwToolbarStyle = .raised,
+@"gtk-toolbar-style": GtkToolbarStyle = .raised,
 
 /// If `true` (default), then the Ghostty GTK tabs will be "wide." Wide tabs
 /// are the new typical Gnome style where tabs fill their available space.
 /// If you set this to `false` then tabs will only take up space they need,
 /// which is the old style.
 @"gtk-wide-tabs": bool = true,
 
-/// If `true` (default), Ghostty will enable Adwaita theme support. This
-/// will make `window-theme` work properly and will also allow Ghostty to
-/// properly respond to system theme changes, light/dark mode changing, etc.
-/// This requires a GTK4 desktop with a GTK4 theme.
-///
-/// If you are running GTK3 or have a GTK3 theme, you may have to set this
-/// to false to get your theme picked up properly. Having this set to true
-/// with GTK3 should not cause any problems, but it may not work exactly as
-/// expected.
-///
-/// This configuration only has an effect if Ghostty was built with
-/// Adwaita support.
-@"gtk-adwaita": bool = true,
```

Comment:
Maybe it would be easier for users if we just blackholed this option and warned in the logs?

Reply:
I'd bet that a lot of users will never see the logs, so I think that we need something noisier than that. I think that options that just silently do nothing will frustrate more people that don't read the release notes. I think that it's going to become increasingly more common to have people that install Ghostty from their package manager and launch it from a GUI desktop icon so they may not even know where to look for the logs.

Reply:
Again, normally I'm very much in the camp of bending over backwards to support backwards compatibility. In this case, we've explicitly warned for a couple releases now that this is coming and that we will be breaking compatibility, so I think in this case, we should hard remove as @jcollie has done and have the config warnings popup come up.